Detectors/TRD/workflow/CMakeLists.txt

```diff
@@ -48,12 +48,6 @@ o2_add_executable(trd-vdrift-exb
                   SOURCES src/trd-calib-workflow.cxx
                   PUBLIC_LINK_LIBRARIES O2::Framework O2::TRDCalibration O2::TRDWorkflow O2::DetectorsCalibration)
 
-
```

Comment:
This should not be removed

Reply:
that was a bad rebase, and by mistake included it then removed it.

Reply:
just clarity the trackreader moved to io

Reply:
But this should not appear in this PR, right?